db/system_keyspace.cc

```diff
@@ -1724,6 +1724,10 @@ future<> system_keyspace::update_tokens(const std::unordered_set<dht::token>& to
 
 future<> system_keyspace::force_blocking_flush(sstring cfname) {
     assert(qctx);
+    if (!qctx->_qp.local().db().get_config().flush_schema_tables_after_modification()) {
+        return make_ready_future<>();
+    }
```

Comment:
> Since a dedicated schema commit log was introduced, a forced
flush of schema tables has become unnecessary. Respect the configuration
option which disables it.

force_blocking_flush() is not used with schema tables. It's used with a subset of (non-schema) system tables (e.g. system.peers). Those tables are not using schema commitlog. So just removing this flush is wrong.

Reply:
This part is now merged.